plugins/arElasticSearchPlugin/config/mapping.yml

```diff
@@ -74,6 +74,16 @@ mapping:
       subject_id: { type: integer, include_in_all: false }
       type_id: { type: integer, include_in_all: false }
 
+  accession_event:
+    _attributes:
+      i18n: true
+      i18nExtra: [property, term]
```

Comment:
Not a big deal but this extra i18n fields are normally used to get the values from inheritance tables, like the actor row for a repository. In the end it works but it's a bit confusing to mix the i18n fields in this case, where they are different entities. It also disables the option to add different configuration to those fields, for example, you may need to make the property value a nested field to search over two fields from the same property (similar to the actor relations issue we discussed a while ago).

In the end, it's probably not an issue in this case, so maybe it's enough to clarify the note in the initial comments to reflect this kind of behavior:

> - i18nExtra: to append i18n fields from other tables (used for inheritance and related resources)

Reply:
In the end, it looks like you only need the term name and the property value. I think `_partial_foreign_types` is a better fit than `i18nExtra` for this fields.

Reply:
Nice... I've reworked it with `_partial_foreign_types` and it's definitely cleaner.